pkgs/desktops/gnome-3/misc/evolution-ews/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv, fetchgit, gnome3, cmake, gettext, intltool, pkgconfig, evolution-data-server
+, sqlite, gtk3, webkitgtk, libgdata, ... }:
+
+stdenv.mkDerivation rec {
+  pname = "evolution-ews";
+  version = "3.36.3";
+  
+  src = fetchgit {
+    url = "https://gitlab.gnome.org/GNOME/${pname}.git/";
+    sha256 = "01qys33nddn27qbg9fk70v4kwkc80gszkfqnwi1grmi42d6igix0";
+    rev = "610facabb38bdf41d993ac0b21866f06fa1fc8dd"; # tag 3.36.3
+  };
+  
+  prePatch = ''
+    substituteInPlace CMakeLists.txt \
+    --replace "add_appdata_file" "#add_appdata_file"
+    substituteInPlace  src/addressbook/CMakeLists.txt \
+    --replace "e-book-backend-ews.c" ""
+    substituteInPlace  src/addressbook/CMakeLists.txt \
+    --replace "e-book-backend-ews-factory.c" ""
+  '';
+  
+  buildInputs = [
+    cmake gettext intltool pkgconfig evolution-data-server gnome3.evolution sqlite libgdata
```

Comment:
1. Tools used for build like `cmake gettext intltool pkg-config` should go to `nativeBuildInputs`.
2. `pkg-config` should be used instead of the alias.